samples/README.md

```diff
@@ -20,7 +21,25 @@ SAGEMAKER_EXECUTION_ROLE_ARN=$(aws iam get-role --role-name ${SAGEMAKER_EXECUTIO
 
 echo $SAGEMAKER_EXECUTION_ROLE_ARN
 ```
-Note down the execution role ARN to use in samples
+Note down the execution role ARN to use in samples.
+
+### S3 Bucket
+Run the following commands to create a S3 bucket which is used by SageMaker service to access and upload data.
+
+```
+export S3_BUCKET_NAME="ack-bucket-$CLUSTER_NAME"
```

Comment:
Add this: Use the region you used during installation unless you are trying cross region. SageMaker resources will be created in this region

```
export AWS_DEFAULT_REGION=<REGION>
export S3_BUCKET_NAME="ack-data-bucket-$AWS_DEFAULT_REGION"
```